.github/workflows/deploy-github-page.yml

```diff
@@ -0,0 +1,106 @@
+name: Build and Deploy
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+    - main
+    
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+    
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Github-page
+            os: ubuntu-24.04
+            emsdk_ver: "3.1.45"
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+        
+      - name: install mamba
+        uses: mamba-org/setup-micromamba@main
+        with:
+          environment-file: environment-wasm-build.yml
+          init-shell: bash
+          environment-name: xeus-r-wasm-build
+
+      - name: Setup default Build Type on *nux
+        if: ${{ runner.os != 'windows' }}
+        run: |
+          echo "ncpus=$(nproc --all)" >> $GITHUB_ENV
+          
+      - name: Setup emsdk
```

Comment:
Maybe we should use the `emscripten_emscripten-wasm32` package from emscripten-forge instead

Reply:
The environment-wasm-build.yaml file needs to be updated also